doc/ref/debug.xml

```diff
@@ -158,6 +160,35 @@ returns the info level of <A>infoclass</A>.
 </Description>
 </ManSection>
 <P/>
+<ManSection>
+<Func Name="ShowUsedInfoClasses" Arg='infoclass'/>
+
+<Description>
+Called with argument <K>true</K>, this makes &GAP; print the info class and level of
+ any executed <Ref Func="Info"/> statement. Calling with the argument <K>false</K> stops this
+ printing.
+
+Each level of each InfoClass is only printed once. The history of printed
```

Comment:
Sorry @ChrisJefferson, I missed just one one.
```suggestion
Each level of each info class is only printed once. The history of printed
```

Reply:
Hi @ChrisJefferson, I saw you did some force pushing since I made this comment this morning, have you perhaps missed it? I'll be happy for the PR to be merged once you make the change (or once you tell me you don't want to make it :wink:).